core/src/main/java/org/bitcoinj/crypto/HDPath.java

```diff
@@ -0,0 +1,120 @@
+package org.bitcoinj.crypto;
```

Comment:
Can you add the license?

Reply:
Fixed in force push: https://github.com/bitcoinj/bitcoinj/compare/7e832e4fd77a5cabd1df09caf3da946c4a140167..df2d543d4204193be42424bbaa6c15687cd82910